biz.aQute.repository/src/aQute/bnd/repository/p2/provider/ArtifactID.java

```diff
@@ -0,0 +1,51 @@
+package aQute.bnd.repository.p2.provider;
+
+class ArtifactID {
+	private final String	id;
+	private final String	version;
```

Comment:
@pkriens This seems a bit of a mess. Artifact uses OSGi Version while IdentityCapability uses bnd Version. Can we somehow avoid this and converge on one Version class in Bnd code?